routes/api.php

```diff
@@ -163,6 +163,13 @@ function () {
 // Routes for data management.
 Route::middleware('admin.data')->group(function () {
     Route::get('dataadmin', 'TeapotController@normalTeapots');
+
+    Route::prefix('admin')->group(function () {
+        Route::get('/airfield/{airfield:code}/stands', 'Admin\\StandAdminController@getStandsForAirfield');
+        Route::get('/airfield/{airfield:code}/stands/{stand}', 'Admin\\StandAdminController@getStandDetails');
```

Comment:
Would it better API-wise for this to be `/stands/{stand_id}`?

Reply:
This auto resolves to the stand model and whatever the key is through the 'SubstituteBindings' middleware, so handles the 404 cases automatically.

Reply:
Putting stand_id would remove the route-model binding also :)

Reply:
Yeah that's fair - just thinking whether from the perspective of the API, `stands` should be a root in its own right, or whether it should always be a subsegment to airfield?